CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+# EQEMU Contribution Guidelines
+
+## Table of contents
+
+- [Contributing pull requests](#contributing-pull-requests)
+- [Code Standards](#code-standards)
+- [Communicating with developers](#communicating-with-developers)
+
+## Contributing pull requests
+
+The preferred way to contribute is to fork the repo and submit a pull request on GitHub. If you need help with your changes, you can always post on the forums or try Discord. You can also post unified diffs (git diff should do the trick) on the Server Code Submissions forum, although pull requests will be much quicker and easier on all parties.
+
+The EQEmu Server project is at its core an emulation project.  It aims to offer feature parity with the latest implemented expansion, which at the time of writing is Gates of Discord. As such, be mindful that PRs that do not reflect the state of live at latest implemented expansion or before might not be desired. Please ensure that:
+
+* The PR makes changes that brings the emulator more in line with the state of live at latest implemented expansion
+* If the PR addresses Server Admin or Server Ops features, they should be general and apply to most or all servers
+* Custom content will not be merged into the Core project
+* Do not submit PRs that are very far from completion
+
+When submitted, contributors are prompted to review/comment on the PR with a goal merge window of 7 days. Please be responsive during this period so as to not let PRs get stuck in limbo.
+
+### Be nice to the Git history
+
+Try to make simple PRs that handle one specific topic. It's better to open 3 different PRs that each address a different issue
+than one big PR with three commits.
```

Comment:
Is this purposefully breaking onto a newline?

Reply:
Not purposely.  I will fix it on next revision. It doesn't show up when rendered anyways as markdown ignores single line breaks